### DISCUSSION TITLE
Business Process Framework

### DISCUSSION THREAD

**Discussion by JORGEGO** [microsoft/semantic-kernel#9428] [General]
At greenYng we find the Business Process Framework very interesting, used both in mobile applications and in the cloud. It is a framework that may have its "rivals" but if we add the ease of adding AI models, it becomes really interesting, but I see that it is in alpha, so I am afraid that it will be a product that will not continue (I think not) and if it will be absorbed by other approaches. Do you suggest that I not get discouraged and continue betting on it? 

Where can I find more information about the evolution of new versions of the product?

Thanks

**Comment by sophialagerkranspandey** [comment]
Thanks for reaching out. We're glad to hear you like the Process Framework. This is not a feature that will be going away and Semantic Kernel is 1.0 and beyond and can be used for production environments. Here's some additional documentation on the Process Framework: https://learn.microsoft.com/en-us/semantic-kernel/frameworks/process/process-framework
